tool/tsh/common/proxy_test.go

```diff
@@ -249,6 +251,60 @@ func TestSSHLoadAllCAs(t *testing.T) {
 	}
 }
 
+// TestWithRsync tests that Teleport works with rsync.
+func TestWithRsync(t *testing.T) {
+	_, err := exec.LookPath("rsync")
+	require.NoError(t, err)
```

Comment:
Alternatively:
```suggestion
	if err != nil {
		t.Skip("this test requires the rsync binary")
	}	
```

Reply:
I vote for the version in this PR. If someone removes the rsync from the Docker image then this test will silently skip. We do not monitor skipped tests, and this already happened in the past.
If we don't want to run this by default, then we can enable/disable it with an environment variable as we do in other cases.